src/app/wallets/ln-send-payment.ts

```diff
@@ -483,7 +493,16 @@ const executePaymentViaLn = async ({
         })
     if (payResult instanceof LnPaymentPendingError) return PaymentSendStatus.Pending
 
-    const settled = await ledgerService.settlePendingLnPayments(paymentHash)
+    const payment = await lndService.lookupPayment({ pubkey, paymentHash })
+    if (payment instanceof Error) return payment
+    payment.paymentRequest = payment.paymentRequest || paymentRequest
+
+    const persistedPayment = await LnPaymentsRepository().update(payment)
+    if (persistedPayment instanceof Error) return persistedPayment
+    const settled = await ledgerService.settlePendingLnPayments({
```

Comment:
is it related to the metadata we want to store? or this is core to the payment logic?

Reply:
This was previous code we had to settle pending transactions by update the `pending` property on the transactions to `true`.

This change is to also include an update for the `_payment` property to add the `ObjectId` value for the payment in the new payments collection.

Reply:
is this to tie the payment from medici to the new `LnPayment` collection? in that case, assuming we have a 1-1 mapping, I would recommend having the `_journal.id` to `LnPayment` instead: this is metadata, so it's better not to overload the medici table with more data, because at scale the more data there is on this collection, the slower the requests like `getBalance()` will run.

Reply:
This was actually just an extra thing and isn't necessary since we already have the mapping via the payment hash. I can remove this to make it simpler.